src/mantine-core/src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import {
+  Box,
+  BoxProps,
+  StylesApiProps,
+  factory,
+  ElementProps,
+  useProps,
+  useStyles,
+  createVarsResolver,
+  Factory,
+  MantineRadius,
+  MantineColor,
+  getRadius,
+  getThemeColor,
+} from '../../core';
+import classes from './Notification.module.css';
+import { Loader } from '../Loader';
+import { Text } from '../Text';
+import { CloseButton } from '../CloseButton';
+
+export type NotificationStylesNames =
+  | 'root'
+  | 'icon'
+  | 'loader'
+  | 'body'
+  | 'title'
+  | 'description'
+  | 'closeButton';
+export type NotificationVariant = string;
+export type NotificationCssVariables = {
+  root: '--notification-radius' | '--notification-color';
+};
+
+export interface NotificationProps
+  extends BoxProps,
+    StylesApiProps<NotificationFactory>,
+    ElementProps<'div', 'title'> {
+  variant?: string;
+
+  /** Called when close button is clicked */
+  onClose?(): void;
+
+  /** Notification line or icon color */
```

Comment:
Props descriptions must be updated from 6.x to be more informative:
- It must include values that can be used – Key of `theme.colors` or any valid CSS color value
- It must include default value for the component – `theme.primaryColor`

Prop description should be: 
"Controls notification line or icon color, key of `theme.colors` or any valid CSS color, `theme.primaryColor` by default"